sanity-io/structured-content-2022
# Editorial articles accessible at /articles/:slug

# Editorial articles accessible at /articles/:slug

## Intent

Solves [this](https://app.shortcut.com/sanity-io/story/16851/update-url-structure-for-editorial-articles) Shortcut Story.

## Testing this PR

1. Verify that all URL references to Editorial Article have been updated
2. Verify that all [available Editorial Articles](http://localhost:3333/desk/article) are accessible on _/articles/:slug_
